src/Workspaces/Core/Portable/Workspace/ProjectSystem/IFileChangeWatcher.cs

```diff
@@ -23,7 +23,7 @@ internal interface IFileChangeWatcher
     /// just creating a single directory watch on the root of a project for source file changes: rather than creating a file watcher
     /// for each individual file, we can just watch the entire directory and that's it.
     /// </remarks>
-    public sealed class WatchedDirectory
+    internal sealed class WatchedDirectory
```

Comment:
Changes to this file should be confirmed with JasonMal as part of #65746

Reply:
Ah yes, that wasn't intended to be made public, thanks for catching that!